recipes-core/psplash/psplash_%.bbappend

```diff
@@ -1,11 +1,15 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 SPLASH_IMAGES:rpi = "file://psplash-raspberrypi-img.h;outsuffix=raspberrypi"
 
-SRC_URI:append:rpi = " file://framebuf.conf"
```

Comment:
Is there any reason not to include this in the existing rules conf file?

Reply:
It's bound to the decision if systemd is activated. From my point it belongs in the same decision tree.

Reply:
the rule has runtime check for systemd so it can go in common rules, however, fb0 might not be same name for framebuffer on all devices

Reply:
Applied. Note the framebuf.conf is pointing to /sys/devices/platform/gpu/graphics/fb0